apps/manager_app/manager_app.h

```diff
@@ -30,7 +30,20 @@
 /*****************************************************************************
  * TYPEDEFS
  *****************************************************************************/
+typedef enum {
+  MANAGER_TASK_SUCCESS = 0,
+  MANAGER_TASK_FAILED,
+  MANAGER_TASK_REJECTED,
+  MANAGER_TASK_INVALID_ARGS,
+  MANAGER_TASK_DECODING_FAILED,
+  MANAGER_TASK_ENCODING_FAILED,
+  MANAGER_TASK_P0_ABORT_OCCURED,
+  MANAGER_TASK_P0_TIMEOUT_OCCURED,
+  MANAGER_TASK_UNKNOWN_QUERY_REQUEST,
+  MANAGER_TASK_INVALID_STATE,
```

Comment:
This a good idea.. I think this is applicable to all flows/applications

Let's discuss this in detail so that we can apply this in other flows also

Reply:
I was unsure of which file to put this in

Please give suggestions @ujjwal-cyph @kartikCypherock

Reply:
these need to be conveyed to host app, so best place is to put these into protobuf.

Reply:
temporarily add a todo for temporary support